examples/pre-push/prevent-force-push-to-master

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+protected_branch='master'
+
+policy='[Policy] Never force push or delete the '$protected_branch' branch! (Prevented with pre-push hook.)'
+
+current_branch=$(git symbolic-ref HEAD | sed -e 's,.*/\(.*\),\1,')
+
+parent_pid=$(ps -oppid= -p $PPID)
+push_command=$(ps -ocommand= -p $parent_pid)
+
+is_destructive='force|delete|\-f'
+
+will_remove_protected_branch=':'$protected_branch
+
+do_exit(){
+  echo $policy
+  exit 1
+}
+
+if [[ $push_command =~ $is_destructive ]] && [[ $push_command =~ $protected_branch ]]; then
```

Comment:
It seems that two conditions can be merged into a single one and then `do_exit` function can be inlined.